tests/integration/features/lock.feature

```diff
@@ -5,57 +5,24 @@ Feature: Lock
     And a working s3
     And a working zookeeper on zookeeper01
     And a working clickhouse on clickhouse01
-    And a working clickhouse on clickhouse02
 
-  @require_version_22.3
```

Comment:
Why was these tests deleted ?

Reply:
All these test are disabled rn (`lock.feature` doesn't included in `ch_backup.featureset`), also they are should not pass with current master branch.

In deleted tests we trying to create backup when [lock](https://github.com/yandex/ch-backup/blob/00ff3e274657bfc8f23659d7bcff6e64b57edfdb/ch_backup/ch_backup.py#L169) is already captured by other process and assuming that there are no backups. But there is a bug, when the lock acquired, we will upload only metadata and the backup will hang in the `creating` state. Finally we will have 1 backup instead of 0. 
This [PR](https://github.com/yandex/ch-backup/blob/00ff3e274657bfc8f23659d7bcff6e64b57edfdb/ch_backup/ch_backup.py#L161) resolving such problem.  
And deleted tests will be added in it, when this PR will be merged.